### DISCUSSION TITLE
Dynamics 365 Finance and Operations support

### DISCUSSION THREAD

**Discussion by michael-polonsky** [AleksandrRogov/DynamicsWebApi#120] [General]
Hello @AleksandrRogov 

Do you think it's possible to use your API and the demo Auth method from https://www.npmjs.com/package/dynamics-web-api#upsert-a-record to work with `Dynamics 365 Finance and Operations`?

**Comment by AleksandrRogov** [comment]
Hi @michael-polonsky ,
This library is for the Microsoft Dataverse. Finance and Operations is a separate application and is not supported.

**Reply by michael-polonsky** [reply]
Thanks for the explanation.
